### PR TITLE
Cu 86b12aru5 notifications list  query data

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
TODO: 
1. Update notifications type categories being saved to the database to match text on filter. 
2. Use correct icons for the notifications.

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b12aru5 Notifications List / Query data](https://app.clickup.com/t/86b12aru5)

File: src/components/notifications/NotificationCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          >
+            {item.title}
+          </Text>
+          <Text style={styles.notificationDetail} level="caption_1">
+            {item.message}
+          </Text>
+        </Stack>
+
+        <Text style={styles.notificationTime} level="caption_1">
+          {displayTime()}
```
Comment (zeckkimon) [inline src/components/notifications/NotificationCard.tsx:51]: When the message is long, the time disappears.
image

**Review by zeckkimon — COMMENTED** [review]
Nice! Just one comment about the date not displaying.

File: src/components/notifications/NotificationCard.tsx (hunk 1 above)
Comment (whamester) [reply src/components/notifications/NotificationCard.tsx:51]: resolved!
It might have something to do with the data in development.

**Pull request merged**